src/modules/colorPicker/ColorPickerUI/Behaviors/MoveWindowBehavior.cs

```diff
@@ -7,21 +11,19 @@ namespace ColorPicker.Behaviors
 {
     public class MoveWindowBehavior : Behavior<Window>
     {
-        public static DependencyProperty LeftProperty = DependencyProperty.Register("Left", typeof(double), typeof(MoveWindowBehavior), new PropertyMetadata(new PropertyChangedCallback(LeftPropertyChanged)));
+        private static readonly DependencyProperty LeftProperty = DependencyProperty.Register("Left", typeof(double), typeof(MoveWindowBehavior), new PropertyMetadata(new PropertyChangedCallback(LeftPropertyChanged)));
 
-        private static void LeftPropertyChanged(DependencyObject d,
-               DependencyPropertyChangedEventArgs e)
+        private static void LeftPropertyChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
```

Comment:
same

Reply:
all that happened here was put stuff on same line

Reply:
sorry my bad!